### PR TITLE
akonadi5: update to 19.04.0.

### PR DESCRIPTION
**Pull request by Hoshpak**

### CODE REVIEW

File: srcpkgs/akonadi5/template [hunk 1]
```diff
@@ -16,7 +16,7 @@ maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="LGPL-2.1-or-later"
 homepage="https://community.kde.org/KDE_PIM/Akonadi"
 distfiles="${KDE_SITE}/applications/${version}/src/akonadi-${version}.tar.xz"
-checksum=f930deaade1cac1488b8af05cc28f4a78a441c34dbe875b673af3423f8a14658
+checksum=c0a2c4259b51293d21487cfe2295df6e8dcfb7ab50eb1a698f531dc7d3253e9a
 conflicts="akonadi"
```
Comment (maxice8) [inline srcpkgs/akonadi5/template]: I think this can be dropped, no ?

Comment (Hoshpak) [reply srcpkgs/akonadi5/template]: If there are no Void installations around somewhere that still use KDE 4, it can be.

**Comment by maxice8** [comment]
On Thu, Apr 18, 2019 at 09:43:12AM -0700, Helmut Pozimski wrote:
> Hoshpak commented on this pull request.
> 
> 
> 
> > @@ -16,7 +16,7 @@ maintainer="Helmut Pozimski <helmut@pozimski.eu>"
>  license="LGPL-2.1-or-later"
>  homepage="https://community.kde.org/KDE_PIM/Akonadi"
>  distfiles="${KDE_SITE}/applications/${version}/src/akonadi-${version}.tar.xz"
> -checksum=f930deaade1cac1488b8af05cc28f4a78a441c34dbe875b673af3423f8a14658
> +checksum=c0a2c4259b51293d21487cfe2295df6e8dcfb7ab50eb1a698f531dc7d3253e9a
>  conflicts="akonadi"
> 
> If there are no Void installations around somewhere that still use KDE 4, it can be.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/11128#discussion_r276744565

We can't know that unless we add PopCorn to base-system and even then we won't be able to reach people that don't update to the point of still use KDE4.

**Comment by Hoshpak** [comment]
So we might as well remove it sooner rather than later, I will do it while fixing cross-compilation.

**Comment by maxice8** [comment]
thank you

**Pull request merged**